src/lib/TranscriptEditor/MediaPlayer/index.module.css

```diff
@@ -43,12 +39,8 @@ video {
 }
 
 /* Mobile devices */
-@media (max-width: 768px) {
+@media (max-width: 769px) {
```

Comment:
It might be best to put in both a min AND max width here. There are some good resources on common breakpoint boundaries for devices: https://responsivedesign.is/develop/browser-feature-support/media-queries-for-common-device-breakpoints/

that way you could have a few, and avoid weird numbers like 769 maybe? also could parameterise them

Reply:
ok, cool, we can pair on this, or just have a chat to decide which once we want to add as first set for this PR.